charts/prefect-worker/values.schema.json

```diff
@@ -40,6 +40,45 @@
       "form": true,
       "additionalProperties": false,
       "properties": {
+        "autoscaling": {
+          "type": "object",
+          "title": "Autoscaling",
+          "description": "autoscaling configuration for the worker",
+          "form": true,
+          "additionalProperties": false,
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "title": "Enabled",
+              "description": "enable autoscaling for the worker",
+              "form": true
```

Comment:
this `form` key doesn't actually adhere to the JSON schema, so im suspecting it doesnt do anything here.  im going to verify and clean this out in a followup PR